playwright-e2e/tests/dsm/cohort-tag.spec.ts

```diff
@@ -1,119 +1,120 @@
-import { test } from '@playwright/test';
-import { login } from 'authentication/auth-dsm';
+import { WelcomePage } from 'dsm/pages/welcome-page';
+import { test } from 'fixtures/dsm-fixture';
 import ParticipantListPage from 'dsm/pages/participant-list-page';
-import HomePage from 'dsm/pages/home-page';
 import ParticipantPage from 'dsm/pages/participant-page/participant-page';
 import CohortTag from 'dsm/component/cohort-tag';
 import { StudyNavEnum } from 'dsm/component/navigation/enums/studyNav-enum';
 import { Navigation } from 'dsm/component/navigation/navigation';
 import * as crypto from 'crypto';
-import { AdditionalFilter } from 'dsm/component/filters/sections/search/search-enums';
-import { WelcomePage } from 'dsm/pages/welcome-page';
 import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
 
-test.describe.skip('Cohort tags', () => {
-  let welcomePage: WelcomePage;
-  let homePage: HomePage;
-  let navigation: Navigation;
-  let cohortTag: CohortTag;
+test.describe('Cohort tags', () => {
+  let shortId: string;
 
-  const studyNames = [StudyEnum.BRAIN, StudyEnum.PANCAN];
-
-  test.beforeEach(async ({ page, request }) => {
-    await login(page);
-    welcomePage = new WelcomePage(page);
-    homePage = new HomePage(page);
-    navigation = new Navigation(page, request);
-    cohortTag = new CohortTag(page);
-  });
+  const studyNames = [StudyEnum.PANCAN];
```

Comment:
Exclude Brain study for now because many participants do not have first and last names. So it will fail to find E2E tests created users.